code/game/jobs/job_controller.dm

```diff
@@ -1,5 +1,10 @@
 var/global/datum/controller/occupations/job_master
 
+#define FREE_ASSISTANTS 2
+
+// The logic requires a shift of 1. The technical reason is that the way it is written, it boils to if (0 > 0) {"reject the assistants"}. Unfortunately, 0 is not > 0.
```

Comment:
// The logic requires a shift of 1. The technical reason is that with the way the system is written, it boils down to if (0 > 0) {"reject the assistants"}. Unfortunately, 0 is not > 0.

Missed some words for clarity. This still confuses me (how is -1 > 0 any better?) but I'm sure it's fine.